client/java-armeria/src/main/java/com/linecorp/centraldogma/client/armeria/ArmeriaCentralDogma.java

```diff
@@ -143,9 +144,8 @@
 
     @Override
     public CompletableFuture<Void> createProject(String projectName) {
+        validateProjectName(projectName);
         try {
-            validateProjectName(projectName);
```

Comment:
I think Trustin intentionally wrapped `validateProjectName(projectName)` with `try...catch` block 
so that users handle an error only in one place, a callback of `CompletableFuture`.
Question: Which one is better for users' experience?

Reply:
We throw an exception directly if the input parameters are invalid without wrapping it even in the method that returns a future throughout our APIs including Armeria.
So I thought we have to throw an exception here as well to ensure consistency. 😄

Reply:
Maybe just me, but I think validating and throwing as quickly as possible is better for user experience.

Reply:
@ikhoon If it's okay with you, let me merge this PR. 😉